app/controllers/users_controller.rb

```diff
@@ -1,5 +1,23 @@
 class UsersController < ApplicationController
-  def index
-    @users = User.all
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    respond_to do |format|
+      if @user.save
+        format.html { redirect_to root_path, notice: 'User was successfully created.' }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+      end
+    end
```

Comment:
respond_to format.html i dont think we need, this is most useful when the endpoint can also be hit with json